File: composeApp/src/androidMain/kotlin/co/jarias/flexapp/ui/screens/bingo/game_list/BingoGameListScreenState.kt
package co.jarias.flexapp.ui.screens.bingo.game_list

import co.jarias.flexapp.domain.Game

data class GroupedBingoGame(
    val lastSession: Game,
    val totalPlayed: Int
)

data class BingoGameListScreenState(
    val games: List<GroupedBingoGame> = emptyList(),
    val isLoading: Boolean = true,
    val errorMessage: String? = null,
    val pendingSetupGameIds: List<Long> = emptyList(),
    val continueToCardSetup: Long? = null,
    val continueToFigureSelection: Long? = null,
    val continueToGamePlay: Long? = null
)

File: iosApp/iosApp/Screens/bingo/game_list/BingoGameListScreenState.swift
import Foundation
import shared

struct BingoGameListScreenState {
    var games: [Game] = []
    var isLoading: Bool = true
    var errorMessage: String? = nil
    var pendingSetupGameIds: [Int64] = []
    var continueToCardSetup: Int64? = nil
    var continueToFigureSelection: Int64? = nil
    var continueToGamePlay: Int64? = nil
}
